README.md

```diff
@@ -1,16 +1,19 @@
-# Ed25519Signature2020 Interoperability Test Suite
+# di-ed25519signature2020-test-suite
+
+> Test Suite for [Ed25519Signature2020](https://w3c-ccg.github.io/lds-ed25519-2020/).
```

Comment:
This URL is giving a 404. Should it be pointing to `https://github.com/w3c/vc-di-eddsa` instead?

Reply:
Yes, specifically, use this URL: https://www.w3.org/TR/vc-di-eddsa/#the-ed25519signature2020-suite